webapp/package.json

```diff
@@ -26,7 +26,7 @@
     "eslint-plugin-react": "7.10.0",
     "jest": "24.8.0",
     "lodash": ">=4.17.13",
-    "webpack": "4.16.1",
+    "webpack": "^4.42.1",
```

Comment:
Could you save the exact version here? Also the `>=` in the `lodash` dependency seems a little odd. Probably fine.

Reply:
Ok. There's been quite a few testing commits, but I was able to get the exact version of webpack with `npm -E audit fix`.